telco-hub/configuration/reference-crs/required/acm/observabilityMCO.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: observability.open-cluster-management.io/v1beta2
+kind: MultiClusterObservability
+metadata:
+  name: observability
+  annotations:
+    mco-disable-alerting: "true"
+spec:
+  advanced:
+    retentionConfig:
+      blockDuration: 2h
+      deleteDelay: 48h
+      retentionInLocal: 3d
+      retentionResolutionRaw: 3d
+  enableDownsampling: false
+  observabilityAddonSpec:
+    enableMetrics: true
+    interval: 300
+  storageConfig:
+    alertmanagerStorageSize: 10Gi
+    compactStorageSize: 100Gi
+    metricObjectStorage:
+      key: thanos.yaml
+      name: thanos-object-storage
+    receiveStorageSize: 25Gi
+    ruleStorageSize: 10Gi
+    storeStorageSize: 25Gi
```

Comment:
Are these sizes "correct" for the retention and downsampling values given above? And are they calculated for maximum deployment (3500 SNO) or core (few clusters large number of nodes) or an average?

Reply:
let me go further about this.